client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -554,6 +559,51 @@ function canItemBeDeleted( item: ResponseCartProduct ): boolean {
 	return ! itemTypesThatCannotBeDeleted.includes( item.product_slug );
 }
 
+function JetpackSearchMeta( { product }: { product: ResponseCartProduct } ): JSX.Element {
+	return (
+		<>
+			<ProductTier product={ product } />
+			<RenewalFrequency product={ product } />
+		</>
+	);
+}
+
+function ProductTier( { product }: { product: ResponseCartProduct } ): JSX.Element | null {
+	const translate = useTranslate();
+	const priceTierList = useSelector( ( state ) =>
+		getProductPriceTierList( state, product.product_slug )
+	);
+
+	if ( isJetpackSearch( product ) && product.current_quantity ) {
+		const tierMaximum = getPriceTierForUnits( priceTierList, product.current_quantity )
+			?.maximum_units;
+		if ( tierMaximum ) {
+			return (
+				<LineItemMeta>
+					{ translate( 'Up to %(tierMaximum)s records', { args: { tierMaximum } } ) }
+				</LineItemMeta>
+			);
+		}
+	}
+	return null;
+}
+
+function RenewalFrequency( { product }: { product: ResponseCartProduct } ): JSX.Element | null {
+	const translate = useTranslate();
+	if ( isMonthlyProduct( product ) ) {
+		return <LineItemMeta>{ translate( 'Renews monthly' ) }</LineItemMeta>;
+	}
+
+	if ( isYearly( product ) ) {
+		return <LineItemMeta>{ translate( 'Renews annually' ) }</LineItemMeta>;
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'renews annually', { context: 'Informative text for renewals in /checkout'} )` *ES Score: 15*

Reply:
These are the same except that the casing is different. I believe it's worthwhile in English to capitalize this sentence.